Doc/library/sqlite3.rst

```diff
@@ -2265,13 +2346,68 @@ can be found in the `SQLite URI documentation`_.
 Explanation
 -----------
 
+.. _sqlite3-transaction-control:
 .. _sqlite3-controlling-transactions:
 
 Transaction control
 ^^^^^^^^^^^^^^^^^^^
 
-The :mod:`!sqlite3` module does not adhere to the transaction handling recommended
-by :pep:`249`.
+:mod:`!sqlite3` offers multiple methods of controlling whether,
+when and how database transactions are opened and closed.
+:ref:`sqlite3-transaction-control-autocommit` is recommended,
+while :ref:`sqlite3-transaction-control-isolation-level`
+is consistent with pre-Python 3.12 versions of the :mod:`!sqlite3` module.
```

Comment:
```suggestion
retains the behavior of pre-Python 3.12 versions of the :mod:`!sqlite3` module.
```

A little clearer phrasing here

Reply:
Or even simpler?

```suggestion
retains pre-Python 3.12 behaviour.
```

Reply:
Yeah, that's better—just missing a definite article:

```suggestion
retains the pre-Python 3.12 behaviour.
```